lib/pbench/server/indexer.py

```diff
@@ -2230,7 +2216,7 @@ def _make_source_individual(self):
         for csvf in self.files:
             assert (
                 csvf["header"][0] == "timestamp_ms"
-            ), "Unexpected time stamp header, '{}'".format(csvf["header"][0])
+            ), f"Unexpected time stamp header, \"{csvf['header'][0]}\""
```

Comment:
This changes the output, although in practice I doubt it matters since we probably don't test it and it'll "never" happen for real. (I actually hate asserts in production code anyway, but that's a different matter.)